docs/features/private-ip.md

```diff
@@ -1,30 +1,53 @@
 # Using Private IP for internal routing
 
-## Pre-requisites
-* Application Gateway with a [Private IP configuration](https://docs.microsoft.com/en-us/azure/application-gateway/configure-application-gateway-with-private-frontend-ip)
+This feature allows to expose the ingress endpoint within the `Virtual Network` using a private IP.
 
-This feature allows to expose the ingress endpoint within the `Virtual Network`.
-To configure the controller to use Private IP for routing, modify the `helm` config by adding `usePrivateIP: true`
+**Pre-requisites**
```

Comment:
GitHub Markdown renders this on the same line as the next. You could add another `\n` or keep the `##` and add another `#`?